packages/oid4vci-holder/src/agent/OID4VCIHolder.ts

```diff
@@ -1016,6 +1008,34 @@ export class OID4VCIHolder implements IAgentPlugin {
     logger.log(`Notification to ${notificationEndpoint} has been dispatched`)
   }
 
+  private async oid4vciGetFederationTrust(args: GetFederationTrustArgs, context: RequiredContext): Promise<boolean> {
+    const { requestData, serverMetadata, trustAnchors } = args
```

Comment:
This method should live in the federation client module probably. If we provide an Issuer string argument there would be nothing left that is specific for oid4vc an we could simple call the method from here using the servermetadata issuer as argument value

Reply:
all this function is is a wrapper around calling the resolve function. so i guess i am not getting what you mean here. the resolve function needs a url, and trustAnchors. it is being called here